docs/accounts/add.md

```diff
@@ -62,35 +60,37 @@ WAN or a private LAN via Wi-Fi. See below for more details if you have trouble.
 Available options are:
 
 * None: This doesn't use any transport security at all.
-* SSL/TLS (if available): SSL/TLS is used but the certificate isn't checked.
-* **SSL/TLS (always)**: SSL/TLS is used and it's checked if the certificate is valid.
-* STARTTLS (if available): The STARTTLS method is used if available; the certificate isn't checked. If STARTTLS is not
+* SSL/TLS (if available): TLS is used but the server's certificate isn't checked for validity.
+* **SSL/TLS (always)**: TLS is used and the server's certificate is checked.
+* STARTTLS (if available): The STARTTLS method is used if available, but the server's certificate isn't checked. If STARTTLS is not
 available, no encryption is used at all.
-* **STARTTLS (always)**: The STARTTLS method is used and the certificate is checked for validity.
+* **STARTTLS (always)**: The STARTTLS method is used and the server's certificate is checked.
 
 **Port**: The port number your provider's SMTP is listening on. This may be *465* or *587*, and in rare cases *25*, 
 depending on the configuration and transport security settings of your server and K-9.
 
 **Require sign-in**: Tells K-9 Mail whether or not it will be expected to authenticate to the server. In almost all 
 cases this needs to remain checked.
 
-**Authentication type**: This specifies which authentication method to use. Available options are:
+**Username**: The username that's needed to authenticate to the SMTP server. This is usually equal to either the
+left-hand side of, or the entire, email address; your mail server operator should have told you what to use as an
+Outgoing username -- and it will commonly be identical to the incoming username, though not always.
+
+**Authentication**: This specifies which authentication method to use. Available options are:
 
-* **AUTOMATIC**: This is the default method that will automatically detect which authentication methods are supported. 
+* **Normal Password**: This is the default method that will automatically detect which authentication methods are supported.
 You shouldn't need to change this.
-* **LOGIN**: This uses the LOGIN authentication method.
-* **PLAIN**: This uses the PLAIN authentication method.
-* **CRAM_MD5**: Use this if your server doesn't support transport security but supports the CRAM_MD5 authentication 
+* **Encrypted Password**: Use this if your server doesn't support transport security but supports the CRAM_MD5 authentication
 method.
-
-**Username**: The username that's needed to authenticate to the SMTP server. This is usually equal to either the 
-left-hand side of, or the entire, email address; your mail server operator should have told you what to use as an 
-Outgoing username -- and it will commonly be identical to the incoming username, though not always.
+* **Client Certificate**: The server expects (only) a client certificate during the TLS connection. The server must not require a password.
```

Comment:
@cketti this and the below section on Client Certificates is new and could do with a check from you please

Reply:
Using a client certificate together with a password is supported in 5.8xx (it wasn't in earlier versions).

Reply:
Yes that's what I thought. If you select "Client Certificate" here, it hides the password box, so this is for the no-password case.

Reply:
Oh, I see. 

Maybe instead of 

> The server expects *only* a client certificate during the TLS connection.

something along the line of

> Use this if the service supports authentication using the client certificate from the TLS connection.

Reply:
Updated